t/14-isotovideo.t

```diff
@@ -62,6 +64,13 @@ subtest 'isotovideo with custom git repo parameters specified' => sub {
     is_in_log('No scripts',        'the repo actually has no test definitions');
 };
 
+subtest 'isotovideo with git refspec specified' => sub {
+    chdir($pool_dir);
+    unlink('vars.json') if -e 'vars.json';
+    isotovideo(opts => "casedir=$data_dir/tests test_git_refspec=deadbeef _exit_after_schedule=1");
+    is_in_log("Checking.*local.*deadbeef", 'refspec in local git repository would be checked out');
```

Comment:
What about the failure case where it dies?

Reply:
Sorry, I don't understand you. Can you please be more specific?

Reply:
This only tests if git succeeds, no?

Reply:
No, this actually fails as there is no "deadbeef" but it only looks if git was triggered correctly regardless if it fails or not.